dev_scripts/env.py

```diff
@@ -307,7 +307,23 @@ def run(
         # We need to retain our UID, because we are mounting the Dangerzone source to
         # the container.
```

Comment:
Does this comment still apply

Reply:
It does, it's just that we now offer a more robust way of retaining it.